scripts/perftest/python/run_perftest.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+# -------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# -------------------------------------------------------------
+
+# TODO:
+# Handel Intercept
+
+import sys
+import argparse
+from functools import reduce
+import os
+from os.path import join
+from utils import get_algo
+from configuration import init_conf
+import logging
+import time
+
+ml_algo = {'binomial': ['MultiLogReg', 'l2-svm', 'm-svm'],
+           'clustering': ['Kmeans'],
+           'multinomial': ['naive-bayes', 'MultiLogReg', 'm-svm'],
+           'regression': ['LinearRegDS', 'LinearRegCG', 'GLM'],
+           'stats': ['Univar-Stats', 'bivar-stats', 'stratstats']}
+
+
```

Comment:
I am not a fan of using the function name `main`. Could you call it something else? maybe something like `perf_test_entry` or something more appropriate.